package.json

```diff
@@ -5,7 +5,7 @@
   "version": "1.8.0",
   "publisher": "CucumberOpen",
   "engines": {
-    "vscode": "^2023.0.0"
+    "vscode": "^1.82.0"
```

Comment:
Looks like `"@types/vscode": "1.85.0",` further down this file also needs a downgrade.